superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -798,6 +807,33 @@ def make_object_info_struct(self, catalog=None):
 
         return obj_str
 
+    def filter_meds(self, outfile, clean=True, min_cutouts=0):
+        ###
+        ### Filter out objects from MEDS that are in fewer 
+        ### than "min_cutouts" exposures
+        
+        if (clean == True) & (min_cutouts > 0):
+        
+            f = fits.open(outfile)
+
+            wb = f[1].data['ncutout'] <= min_cutouts
+
+            print(f'{len(f[1].data[wb])}/{len(f[1].data)} objects with fewer than {min_cutouts} found')
+            print(f'removing {len(f[1].data[wb])}/{len(f[1].data)} objects; saving original to full.meds')
+
+            wg = f[1].data['ncutout'] > min_cutouts
+            f[1].data = f[1].data[wg]
+            
+            if (min_cutouts > 3):
```

Comment:
It's confusing and should be removed.

Reply:
I agree with saving the full MEDS file just in case something goes wrong, though this could be simplified by just using an f-string: `cmd_str = f'mv {outfile} {full_name}'`, then no `format` is needed.

Reply:
Also just to be clear, I'd have this done if `min_ncutout` is ever > 0, not just > 3.